eng/versioning/version_client.txt

```diff
@@ -63,7 +63,7 @@ com.azure:azure-communication-common;1.1.5;1.2.0-beta.1
 com.azure:azure-communication-common-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-sms;1.1.4;1.2.0-beta.1
 com.azure:azure-communication-identity;1.1.11;1.2.0-beta.2
-com.azure:azure-communication-phonenumbers;1.0.12;1.1.0-beta.9
+com.azure:azure-communication-phonenumbers;1.0.13;1.1.0-beta.9
```

Comment:
We should get this PR merged - https://github.com/Azure/azure-sdk-for-java/pull/29974/files

Reply:
That PR was for reviewing purposes of a patch release branch. We should not merge it as it would get rid of all beta work on `main`.